webapi/CopilotChat/Hubs/MessageRelayHub.cs

```diff
@@ -39,17 +39,18 @@ public async Task AddClientToGroupAsync(string chatId)
     /// <summary>
     /// Sends a message to all users except the sender.
     /// </summary>
-    /// <param name="chatId">The ChatID used as group id for SignalR.</param>
+    /// <param name="chatId">The chat ID used as group id for SignalR.</param>
+    /// <param name="userId">The user ID used as group id for SignalR.</param>
```

Comment:
Why is both chatId and userId user for groupId here?
`userId` make more sense as `senderId` or `sourceId` since it's overloaded with the "userId" property that already exists in the ChatMessage object (This was confusing when trying to follow the CreateBotMessageOnClient flow in ChatSkill)

```suggestion
    /// <param name="userId">The user ID of the source of the message.</param>
```